docs/aggregate.md

```diff
@@ -8,7 +8,7 @@ Groups rows by the group_by items applying aggregations functions for the result
 
 |   Argument   |    Type     |                                                             Description                                                             | Is Optional |
 | ------------ | ----------- | ----------------------------------------------------------------------------------------------------------------------------------- | ----------- |
-| group_by     | column_list | Columns to group by                                                                                                                 |             |
```

Comment:
@alexwolf22 I didn't make these changes but they appear to be different as a result of some automated generations between my view and main.  Can you offer some insights about what this should look like here?

Reply:
Yeah since you set `group_by` transform param to True in the YAML, it added that to the Markdown.

Markdown file seems right here.